setup-environment-internal

```diff
@@ -388,6 +388,9 @@ SDKMACHINE ?= "${SDKMACHINE}"
 INHERIT += "rm_work"
 EOF
 
+# Because we are (re-)generating the build conf files, ensure we remove
+# the ACCEPT_EULA_${MACHINE} file to present the EULA to the user
+rm -rf ACCEPT_EULA_*
```

Comment:
Why? If the user has accepted the EULA then why do we need to ask again?

Reply:
> Why? If the user has accepted the EULA then why do we need to ask again?

Mostly because the auto.conf will be regenerated. And currently, the `mbl_eula_handle` function ignores showing the EULA if the `ACCEPT_EULA_*` file is present.